### ISSUE TITLE
Evaluation Script

### ISSUE THREAD

**Issue by SuryaKrishna02** [SuryaKrishna02/maya-dataset-creation#2]
- Given the prompt, it should evaluate the prompt for the given language or all languages using BLEU or METEOR Score to find how good the prompt is.
- Feel free to explore any BLUE and METEOR alternatives.

**Comment by rematchka** [comment]
@SuryaKrishna02 Could you explain more about the task?

**Comment by SuryaKrishna02** [comment]
@rematchka Updated the Issue description. If you have any more doubts, we can discuss here or you can directly DM on discord and discuss over a short call.

If you feel something can be done in a more better way, Please let me know :)

**Comment by Karthikreddyk99** [comment]
Yeah lets take the discussion of this topic to Discord and comeback with it.
